test/src/test/java/org/corfudb/integration/CorfuReplicationClusterConfigIT.java

```diff
@@ -1626,16 +1644,18 @@ public void testBackupRestoreWorkflow() throws Exception {
             }
         }
 
+        // Note that the topologyConfigId on Sink will be checked, as the backup cluster will be brought up as new Source
+        // and the new topologyConfigId will not appear on previous Source cluster
+        long persistedTopologyConfigId = getSinkPersistedTopologyConfigId();
         // Change the topology - brings up the backup cluster
         updateTopology(activeCorfuStore, DefaultClusterManager.OP_BACKUP);
         log.info("Change the topology!!!");
-
-        TimeUnit.SECONDS.sleep(shortInterval);
-
+        while (persistedTopologyConfigId >= getSinkPersistedTopologyConfigId()) {
```

Comment:
Why do we check the topology config id on the Sink?  How will its topology config id change if OP_BACKUP was written and processed on the Source cluster?